IBAnimatableApp/BlurEffectViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  BlurEffectViewController.swift
+//  IBAnimatableApp
+//
+//  Created by jason akakpo on 27/07/16.
+//  Copyright © 2016 Jake Lin. All rights reserved.
+//
+
+import UIKit
+import IBAnimatable
+
+class BlurEffectViewController: UIViewController {
+  
+  @IBOutlet weak var imageView: AnimatableImageView!
+  @IBOutlet weak var pickerView: UIPickerView!
+  let opacityValues = ParamType.number(min: 0.0, max: 1.0, interval: 0.1, ascending: false, unit: "")
+  lazy var values: [String] = {
+    var values = ["none", "extraLight", "light", "dark"];
+    if #available(iOS 10.0, *) {
+      values.append(contentsOf: ["prominent", "regular"])
+    }
+    return values
+  }()
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    pickerView.delegate = self
```

Comment:
Oh I see. 
It's important to have some code consistency ; I will do it your way.